docs/en/observability/install-apm.asciidoc

```diff
@@ -0,0 +1,56 @@
+[chapter, role="xpack"]
+[[observability-instrument-apps]]
+= Instrument applications
+
+Elastic APM monitors software services and applications in real time, collects unhandled
+errors and exceptions, and automatically picks up basic host-level metrics and agent specific metrics.
+
+
+[NOTE]
+=====
+If you haven’t already, you need to install {es} for storing and searching your data, {kib} for
+visualizing and managing it, and APM Server. For more information, see <<install-observability,Get started>>.
+=====
+
+[[install-apm-agents]]
+== Step 1: Install APM agents
+:leveloffset: -1
+include::{apm-repo-dir}/guide/install-and-run.asciidoc[tag=apm-agent]
+:leveloffset: +1
+[[configure-apm-data]]
+== Step 2: Configure APM
+
+Now that you're up and running with Elastic APM, you may want to adjust some configuration settings.
+Luckily, there are many different ways to tweak and tune the Elastic ecosystem to adapt it to your needs.
+
+[[configure-apm-agents]]
+=== Configure APM agents
+:leveloffset: -1
+include::{apm-repo-dir}/guide/install-and-run.asciidoc[tag=configure-agents]
+:leveloffset: +1
+[[configure-elastic-cloud]]
+
+[[quick-start-environment]]
+=== Quick start development environment
```

Comment:
The more I look at this, the more I dislike having this content here. It makes the page too long. It also interrupts the original setup method by coming before the final step 3

Reply:
No problem. I removed this section.